lib/mozart_fetcher/timeout_parser.ex

```diff
@@ -6,6 +6,13 @@ defmodule MozartFetcher.TimeoutParser do
     query(uri.query)
   end
 
+  def max_timeout(components) do
+    components
```

Comment:
Same discussion:
https://github.com/bbc/mozart-fetcher/pull/40#discussion_r286063590

Reply:
See PR for this:
https://github.com/bbc/mozart-fetcher/pull/42